upgrade_test.py

```diff
@@ -23,16 +23,17 @@
 from functools import wraps, cache
 from typing import List
 
-from argus.db.db_types import PackageVersion
 from pkg_resources import parse_version
+from argus.db.db_types import PackageVersion
```

Comment:
why this is moved ?

Reply:
change it back to the same place.

Reply:
Pylint alerted it was the wrong import order